s3_to_sfts/s3_to_sfts.py

```diff
@@ -149,34 +149,44 @@ def report(data):
         f'ended at: {yvr_dt_end.strftime("%Y-%m-%d %H:%M:%S%z (%Z)")}, '
         f'elapsing: {yvr_dt_end - yvr_dt_start}.')
     print(f'\nItems to process: {data["objects"]}')
-    print(f'Objects successfully processed to s3: {data["objects_processed"]}')
-    print(f'Objects unsuccessfully processed to s3: {data["objects_not_processed"]}')
-    print(f'Objects successfully processed to sfts: {len(data["s3_good_list"])}')
-
-    # Print all objects loaded into s3/good
-    if data['s3_good_list']:
-        print(f'\nObjects loaded to S3 /good:')  
-        for i, item in enumerate(data['s3_good_list'], 1):
-            print(f"\n{i}: {item}")
-
-    # Print all objects loaded into s3/bad
-    if data['s3_bad_list']:
-        print(f'\nObjects loaded to S3 /bad:')
-        for i, item in enumerate(data['s3_bad_list'], 1):
-            print(f"\n{i}: {item}")
+    print(f'Objects successfully processed to sfts: {len(data["sfts_processed_list"])}')
+    print(f'Objects unsuccessfully processed to sfts: {len(data["sfts_not_processed_list"])}')
+    print(f'Objects successfully processed to s3: {len(data["s3_processed_list"])}')
+    print(f'Objects unsuccessfully processed to s3: {len(data["s3_not_processed_list"])}')
```

Comment:
Change "Objects unsuccessfully processed to sfts" to "Objects failed processing to sfts".

Reason - It will be easier to distinguish between lines and more explicit